routes/server.go

```diff
@@ -1419,6 +1422,13 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.LockedBalanceEntries,
 			PublicAccess,
 		},
+		{
+			"CreateAtomicTxnsWrapper",
+			[]string{"GET"},
```

Comment:
I think you want post and options.
```suggestion
			[]string{"POST", "OPTIONS"},
```

Reply:
Yes you're definitely right. Will correct this.